### DISCUSSION TITLE
Windows Defender & Malwarebytes quarantined v22621.3880.66.5

### DISCUSSION THREAD

**Discussion by Panix-Button** [valinet/ExplorerPatcher#3664] [General]
Both Windows Defender and Malwarebytes Premium flagged v22621.3880.66.5 as Malware/PUP/HackTool and the setup file was quarantined. Windows Defender flagged it first so thinking it was a false positive, I turned off real time protection, then MB quarantined it too. I'm sure it's just a false positive but until further input I won't turn off MB until I'm sure nothing is wrong.

**Comment by pyrates999** [comment]
read release notes here on what you need to do with AV in order to install EP: https://github.com/valinet/ExplorerPatcher/releases

**Reply by jdpm1991** [reply]
> read release notes here on what you need to do with AV in order to install EP: https://github.com/valinet/ExplorerPatcher/releases

It still says it's a virus when downloading the exe

**Reply by pyrates999** [reply]
you didn't read the release notes

**Reply by pyrates999** [reply]
mentioned here: https://github.com/valinet/ExplorerPatcher/issues/3228

**Reply by floppyedonkey** [reply]
You reference a post from May... thank god I stopped using this addon, it has been problem ridden.

**Reply by Verbo84** [reply]
Use Terminal (Admin). It didn't work when I tried Windows PowerShell (Admin).
